firestore.rules

```diff
@@ -25,8 +25,12 @@ service cloud.firestore {
         && request.auth.token.platform_id == resource.data.platform_id;
     }
 
-    // Check if answer has been explicitly shared by the owner of the answer (shared_with: context) and
-    // if the user who is trying to read the answer is member of the same context as the author.
```

Comment:
Wasn't that true anymore?

Reply:
Good catch.  I created these rules on `report-service-dev` for development but for this PR I diffed the `report-service-pro` and `report-service-dev` rules to ensure there were no manually applied changes on pro and that comment was on pro and not dev but I did not copy over the comment.  I've added it back.